build/bin/vars

```diff
@@ -10,13 +10,13 @@ tmp_dir=${root_dir}/tmp
 # * pmm_release to contain only pre-release part (may be empty): 'alpha3'.
 pmm_branch=$(git rev-parse --abbrev-ref HEAD)
 full_pmm_version=$(cat ${root_dir}/VERSION)
-if [[ ${pmm_branch} != 'PMM-2.0' ]]; then
+if [[ ${pmm_branch} != "release"* && ${pmm_branch} != "PMM-2.0" && ${pmm_branch} != "main" ]]; then
```

Comment:
&& or ||

Reply:
if branch is not release and it is not PMM-2.0 and it is not main - long name name. Or would be used if inside of if block would be a short version but else would be long....

Reply:
oh yes, I got confused, thats right.